view/frontend/templates/footer/zopim.phtml

```diff
@@ -13,12 +12,8 @@
             ], function ($) {
                 $(document).ready(function () {
 
-                    var cookieName = "<?php echo $block->config->getExpectedCookieName() ?>"
-                    var cookieValue = "<?php echo $block->config->getExpectedCookieValue() ?>"
-                    
-                    console.log(cookieName);
-                    console.log(cookieValue);
-                    console.log($.cookie(cookieValue));
+                    var cookieName = "<?= $block->escapeHtmlAttr($block->config->getExpectedCookieName()) ?>"
```

Comment:
We should use the method `escapeHtml` here instead of `escapeHtmlAttr`.

Reply:
@Tjitse-E should this not be escapeJS? https://devdocs.magento.com/guides/v2.4/extension-dev-guide/xss-protection.html

Reply:
@esteban-serfe  agreed, let's use `escapeJS`. Can you change this? Then I will merge this PR and create a new release.

Reply:
@Tjitse-E Done